drivers/I2C.h

```diff
@@ -117,8 +117,8 @@ class I2C {
      *  @param repeated Repeated start, true - do not send stop at end
      *
      *  @returns
-     *      0 or non-zero - written number of bytes,
-     *      negative - I2C_ERROR_XXX status
+     *       0 on success (ack),
+     *   non-0 on failure (nack)
```

Comment:
I recall this was updated not that long ago. Is a positive number really a failure?

This came from i2c HAL write that specifies the following:

```
`/** Blocking sending data
 *
 *  @param obj     The I2C object
 *  @param address 7-bit address (last bit is 0)
 *  @param data    The buffer for sending
 *  @param length  Number of bytes to write
 *  @param stop    Stop to be generated after the transfer is done
 *  @return 
 *      zero or non-zero - Number of written bytes
 *      negative - I2C_ERROR_XXX status
 */
int i2c_write(i2c_t *obj, int address, const char *data, int length, int stop);
```

How many targets do follow this new update? how does it map to the current one?